packages/package-json-lint/tests/jest/terra-cli/package-json-lint/index.test.js

```diff
@@ -0,0 +1,32 @@
+jest.mock('../../../../src/lint');
+
+const yargs = require('yargs');
+const lint = require('../../../../src/lint');
+const PackageJsonLintCommand = require('../../../../src/terra-cli/package-json-lint');
+
+describe('package-json-lint', () => {
+  it('declares the package-json-lint terra-cli command with proper top level help', async () => {
+    const parser = yargs.command(PackageJsonLintCommand).scriptName('terra').help();
+    const helpOutput = await new Promise((resolve) => {
+      parser.parse('--help', (err, argv, output) => {
+        resolve(output);
+      });
+    });
+    expect(helpOutput).toMatchSnapshot();
+  });
+
+  it('declares the package-json-lint terra-cli command with proper command level help', async () => {
+    const parser = yargs.command(PackageJsonLintCommand).scriptName('terra').help();
+    const helpOutput = await new Promise((resolve) => {
+      parser.parse('prepare-for-release --help', (err, argv, output) => {
```

Comment:
What is this testing for this component?

Reply:
It's testing the setup here: https://github.com/cerner/terra-toolkit/blob/create-package-json-lint-command/packages/package-json-lint/src/terra-cli/package-json-lint/index.js

Right now that's not so much, but if/when we add command line configs to the mix it will test that config.

Reply:
ah okay. I wasn't sure with the prepare-for-release thanks for clarification

Reply:
Now I understand your concern. Copy and paste error.  Oops.  Good catch! https://github.com/cerner/terra-toolkit/pull/555/commits/99bb1d7d992801941908a27fdf4f290d0d68e386